packages/test/local-server-stress-tests/src/stressDataObject.ts

```diff
@@ -347,6 +347,8 @@ export const createRuntimeFactory = (): IRuntimeFactory => {
 			return this;
 		},
 		instantiateRuntime: async (context, existing) => {
+			// This can be removed or scoped to options passed to specific data stores once we support squashing more widely.
+			context.options.allowStagingModeWithoutSquashing = true;
```

Comment:
this isn't necessary right? sine we switch to allowStagingModeWithoutSquashing  by default and will go away?